test/statscollector/DefaultStatsCollector.test.ts

```diff
@@ -414,50 +343,111 @@ describe('DefaultStatsCollector', () => {
             state: 'succeeded',
           })
         ).to.be.true;
-        statsCollector.stop();
-      });
-
-      it('checks Firefox raw metrics', () => {
-        setUserAgent(FIREFOX_USERAGENT);
-        statsCollector = new DefaultStatsCollector(audioVideoController, logger, browser, interval);
-        statsCollector.start(signalingClient, new DefaultVideoStreamIndex(logger));
         expect(
           statsCollector.isValidStandardRawMetric({
             type: 'inbound-rtp',
           })
         ).to.be.true;
-        statsCollector.stop();
-      });
-
-      it('checks Firefox raw metrics when the current browser version matches the expected major version', () => {
-        setUserAgent(
-          'Mozilla/5.0 (Macintosh; Intel Mac OS X 10.12; rv:68.0) Gecko/20100101 Firefox/66.0'
-        );
-        statsCollector = new DefaultStatsCollector(audioVideoController, logger, browser, interval);
-        statsCollector.start(signalingClient, new DefaultVideoStreamIndex(logger));
         expect(
           statsCollector.isValidStandardRawMetric({
             type: 'outbound-rtp',
-            isRemote: false,
           })
         ).to.be.true;
-        statsCollector.stop();
-      });
-
-      it('checks Firefox raw metrics if the current browser version is lower than the expected major version', () => {
-        setUserAgent(
-          'Mozilla/5.0 (Macintosh; Intel Mac OS X 10.12; rv:68.0) Gecko/20100101 Firefox/64.0'
-        );
-        statsCollector = new DefaultStatsCollector(audioVideoController, logger, browser, interval);
-        statsCollector.start(signalingClient, new DefaultVideoStreamIndex(logger));
         expect(
           statsCollector.isValidStandardRawMetric({
-            type: 'outbound-rtp',
-            isRemote: false,
+            type: 'remote-inbound-rtp',
+          })
+        ).to.be.true;
+        expect(
+          statsCollector.isValidStandardRawMetric({
+            type: 'remote-outbound-rtp',
           })
         ).to.be.true;
+
         statsCollector.stop();
       });
+
+      describe('checks browser version', () => {
+        it('checks raw metrics if the current Firefox browser version is equal to the expected major version', () => {
+          setUserAgent(
+            'Mozilla/5.0 (Macintosh; Intel Mac OS X 10.12; rv:68.0) Gecko/20100101 Firefox/66.0'
```

Comment:
What does all this tests do? Firefox min version support is 75.

Reply:
Removed the Firefox version check in `DefaultStatsCollector` as well as `browserBehavior`